docs/services.md

```diff
@@ -6,7 +6,7 @@ using the Docker Engine API. Note that the engine needs to be part of a
 
```

Comment:
Oops – I think this file is redundant now. The content has been moved to docstrings and `api.rst`. There is also a file called `services.rst` which will be displayed instead of this file.

We can just delete it, I think.